pages.php

```diff
@@ -32,6 +32,32 @@
     <head>
     </head>
     <body>
-        <?php /* Use PHP's include() method to render the correct page content in here */ ?>
+        <?php /* Use PHP's include() method to render the correct page content in here */ 
+            echo'
+                <a href="/pages.php?page=home"> ++Home++ </a>
+                <a href="/pages.php?page=about"> ++About++ </a>
+                <a href="/pages.php?page=interests"> ++Interests++ </a>';
+                
+                function about (){
+                    echo"";
```

Comment:
@kko31623 you should use PHP's `include` function to include the content from the external files (in the `/pages` directory). You use `include` like this:

```php
  <?php
    include( dirname(__FILE__) . '/pages/filename.php');
  ?>
```